solution/index.js

```diff
@@ -0,0 +1,222 @@
+let Tasks = {
+    'todo': [],
+    'in-progress': [],
+    'done': []
+};
+
+// maping the ul names to make genrated code 
+const mapper = {
+    '1': 'todo',
+    '2': 'in-progress',
+    '3': 'done'
+}
+
+let mouseHover;
+let onAlt = false;
+// making on keydown and key up global 
+document.onkeydown = (e) => {
+    if (e.key === 'Alt') {
```

Comment:
There's a better way to check if alt is pressed. Read about keyboard events

Reply:
Fixed